app/views/users/logix/index.html.erb

```diff
@@ -32,6 +32,11 @@
   <div class ="row">
     <h2 class="card-title"><%= @user.name%> - Projects | <a href=  "<%= profile_path(@user)%>"  target="_blank"> View Profile </a> | <a href=  "<%= user_favourites_path(@user)%>"  target="_blank"> View Favourites </a> </h2>
   </div>
+  <% if @user.projects.blank? %>
+    <div class ="noprojects" style="position: absolute;top: 30%;left: 30%;">
+      <h5>You dont have any projects. Create one <a href=  "/simulator"  target="_blank"> here </a></h5>
```

Comment:
Use `link_to` helper, Open in same window, not new window

Reply:
@tachyons won't removing ````taget="_blank"```` solve the problem?

Reply:
@tachyons target="_self" will also solve  problem